### ISSUE TITLE
Getting PipelineRun  instead of pipelineActivity

### ISSUE THREAD

**Issue by rajatgupta24** [jenkins-x/jx#8308]
# Describe
On canceling the PipelineActivity, I was getting cancelled PipelineRun.

I knew my PipelineActivity would fail, the timeout was set 12hrs, so I stopped it.
I run `jx pipeline stop`

# Output
![image](https://user-images.githubusercontent.com/55191777/179771906-400a1bf7-977d-459b-b421-9d32f33236b3.png)

**Comment by ankitm123** [comment]
This is expected. as the underlying resource is a pipeline run, are u concerned about the wording?

**Comment by rajatgupta24** [comment]
Yes, I thought it would be PipelineActivity, as PipelineRuns is actually PipelineActivity in the Jenkins X.
